cloudfoundry-operations/src/main/java/org/cloudfoundry/operations/applications/ApplicationManifestUtils.java

```diff
@@ -125,21 +149,32 @@ public static void write(OutputStream out, List<ApplicationManifest> application
         }
     }
 
-    private static <T> void as(Map<String, Object> payload, String key, Function<Object, T> mapper, Consumer<T> consumer) {
+    private static <T> void as(Map<String, Object> payload, String key, Map<String, String> variables, Function<Object, T> mapper, Consumer<T> consumer) {
         Optional.ofNullable(payload.get(key))
+            .map(o -> {
+                if(o instanceof String) {
+                    Matcher m = FIND_VARIABLE_REGEX.matcher(((String) o));
```

Comment:
```suggestion
                    Matcher m = FIND_VARIABLE_REGEX.matcher((String) o);
```

I don't believe you need the extra `(..)`.